CondTools/SiStrip/plugins/SiStripDB2Tree.cc

```diff
@@ -0,0 +1,347 @@
+// -*- C++ -*-
+//
+// Package:    CondTools/SiStrip
+// Class:      SiStripDB2Tree
+//
+/**\class SiStripDB2Tree SiStripDB2Tree.cc CondTools/SiStrip/plugins/SiStripDB2Tree.cc
+
+ Description:
+     Converts several SiStrip Payloads (Noise, Peds, APVGain, etc.) into a TTree
+*/
+//
+// Original Author:  Mauro Verzetti
+// Modified by:      Marco Musich
+//
+
+// system include files
+#include <map>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+// user include files
+#include "CalibFormats/SiStripObjects/interface/SiStripQuality.h"
+#include "CalibTracker/Records/interface/SiStripQualityRcd.h"
+#include "CalibTracker/SiStripCommon/interface/SiStripDetInfoFileReader.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+#include "CondFormats/DataRecord/interface/SiStripApvGainRcd.h"
+#include "CondFormats/DataRecord/interface/SiStripCondDataRecords.h"
+#include "CondFormats/DataRecord/interface/SiStripNoisesRcd.h"
+#include "CondFormats/DataRecord/interface/SiStripPedestalsRcd.h"
+#include "CondFormats/SiStripObjects/interface/SiStripApvGain.h"
+#include "CondFormats/SiStripObjects/interface/SiStripLatency.h"
+#include "CondFormats/SiStripObjects/interface/SiStripNoises.h"
+#include "CondFormats/SiStripObjects/interface/SiStripPedestals.h"
+#include "DataFormats/Common/interface/DetSetNew.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+#include "DataFormats/SiStripCommon/interface/ConstantsForHardwareSystems.h" /* for STRIPS_PER_APV*/
+#include "DataFormats/SiStripDetId/interface/SiStripDetId.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#define LOGERROR(x) edm::LogError(x)
+#define LOGINFO(x) edm::LogInfo(x)
+#define LOGDEBUG(x) LogDebug(x)
+
+// ROOT includes
+#include "TNamed.h"
+#include "TObjString.h"
+#include "TText.h"
+#include "TTree.h"
+
+//**********************************************//
+// Auxilliary class
+//**********************************************//
+class RecordInfo : public TNamed {
+public:
+  RecordInfo(const char* record, const char* tag) : TNamed(record, tag) {}
+
+  void printInfo() const { LOGINFO("RecordInfo") << GetName() << " " << GetTitle() << std::endl; }
```

Comment:
Since `LOGINFO` is essentially `edm::LogInfo`, I believe the `<< std::endl` is not necessary

Reply:
i'd rather pass this time. thank you.

Reply:
addressed in the last push, though.